src/metabase/api/database.clj

```diff
@@ -337,7 +337,9 @@
       true                         add-expanded-schedules
       true                         (get-database-hydrate-include include)
       include-editable-data-model? check-db-data-model-perms
-      mi/can-write?                secret/expand-db-details-inferred-secret-values)))
+      mi/can-write?                (cond->
```

Comment:
this `cond->` looks wrong. `secret/expand-db-details-inferred-secret-values` is a function but here is used as an always truthy value to always `(assoc :can-manage true)`. i think you just want a regular `->` through these two functions.

Reply:
Good catch, thank you.